llarp/constants/platform.hpp

```diff
@@ -0,0 +1,65 @@
+#pragma once
+
+/// namespace for platform feature detection constexprs
+namespace llarp::platform
+{
+  ///  are we linux  ?
+  constexpr bool is_linux =
```

Comment:
These need `inline` added to avoid potential duplicate symbol clashes:
```suggestion
  inline constexpr bool is_linux =
```